code/_onclick/item_attack.dm

```diff
@@ -58,19 +60,27 @@
 		user.visible_message("<span class='danger'>[user] has hit [src] with [I]!</span>", "<span class='danger'>You hit [src] with [I]!</span>")
 
 /mob/living/attacked_by(obj/item/I, mob/living/user)
-	if(user != src)
-		user.do_attack_animation(src)
-	if(send_item_attack_message(I, user))

```

Comment:
pretty much nothing changed here, just moving things around because I didn't like how the code used "if(send_item_attack_message())", the reader has no idea why this proc returns true or not. It was just there to not have to check if(I.force) again.